File: ABC/ABC114/B/main.java
import java.util.*;

public class Main {

  public static void main (String[] args) {

    Scanner sc = new Scanner(System.in);
    
    String S = sc.next();

    sc.close();

    String[] s = S.split("");

    int d = 753;

    for (int i = 0; i <= s.length - 3; i++) {

      int v = Integer.valueOf(s[i] + s[i+1] + s[i+2]);

      int d1 = Math.abs(753 - v);
      
      d = Math.min(d, d1);

    }

    System.out.println(d);

  }

}

File: ABC/ABC114/B/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

  string S;
  cin >> S;
  
  int ans = 1000;

  for (int i = 0; i < S.size() - 2; i++) {
    int a_ = (S[i] - '0') * 100 + (S[i+1] - '0') * 10 + (S[i+2] - '0');
    ans = min(ans, abs(a_ - 753));
  }
  
  cout << ans << endl;

}